docs/MatrixOne/Develop/import-data/bulk-load/bulk-load-overview.md

```diff
@@ -0,0 +1,14 @@
+# Bulk Load Overview
+
+Bulk load data is supported in MatrixOne. Using the MatrixOne database, according to different data volumes, file types, or additional data storage locations, you can choose other import methods to improve your work efficiency.
+
+## Importing the *.csv* or jsonlines format data
+
+MatrixOne currently mainly supports importing two data formats:
+
+- Support importing *.csv* format. For more information, see [Import *.csv* format data](load-csv.md).
+- Support importing *.jl* format, jsonlines. For more information, see [Import jsonlines format data](load-jsonline.md).
+
+## Importing data from S3
```

Comment:
this description is incomplete. loading from s3 is one location, the other is from local file. you need to specify both and give examples.